src/test_l1_deepSleepMgr.c

```diff
@@ -161,8 +161,12 @@ void test_l1_deepSleepMgr_negative_PLAT_DS_INIT (void)
     UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_SUCCESS);
 
     // Variation 02: Call PLAT_DS_INIT() again and expect it to be already initialized
-    result = PLAT_DS_INIT();
-    UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_ALREADY_INITIALIZED);
+     result = PLAT_DS_INIT();	
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.